frontend/cypress/integration/dashboard-page.spec.ts

```diff
@@ -24,51 +22,54 @@ context("Dashboard", () => {
     cy.get(
       ".card-area .dashboard-card.node-card .dashboard-card-header"
     ).contains("Nodes");
-    cy.get(".card-area .dashboard-card.node-card a .long-card-cell.href-cell", {
```

Comment:
Tests in this file with `each` didn't actually work, if one flips a switch on `loading` - the tests still pass.
I made them a little more sensible though I suggest to migrate to something user-friendlier - like playwright or plain puppeteer.